build.py

```diff
@@ -67,8 +67,8 @@ def convertToHtml(text):
 
     data_home["alumni_description"] = convertToHtml(data_home["alumni_description"])
 
-    for community in communities:
-        community["desc"] = convertToHtml(community["desc"])
+    # for community in communities:
+    #     community["desc"] = convertToHtml(community["desc"])

```

Comment:
Why did you comment this out?
We would still need to convert the markdown discription into HTML supported text

Reply:
Also for the conference descriptions

Reply:
Previously community.desc contained a string with markdown containing the description paragraph as well as text-based links and link heading. But we restructured the JSON to separate out the links so we could loop through them in the template and turn them into icon links. Having taken out the links, community.desc now just contains the description paragraph. Instead of converting it from markdown to HTML we placed the string inside <p> tags in the template, making this markdown to HTML conversion redundant - hence the comment out, but perhaps we should delete it instead.

Reply:
But if anyone wants to use some Markdown formatting in their community or conference description, it would still be useful. 

And it is called once only during building the site, so it's not like it'll introduce any performance overhead to keep it. 

So, I think it is better to keep the convertToHtml call just in case.

Reply:
ok, will do

Reply:
Changes complete